src/index.js

```diff
@@ -20,15 +20,88 @@ function createElement(tagName, props, ...children) {
   return element;
 }
 
+const state = {
+  result: '',
+  numbers: [],
+  operator: '',
+};
+
+const calculator = {
+  plus: (...numbers) => Number(numbers[0]) + Number(numbers[1]),
+  minus: (...numbers) => Number(numbers[0]) - Number(numbers[1]),
+  multiply: (...numbers) => Number(numbers[0]) * Number(numbers[1]),
+  divide: (...numbers) => Number(numbers[0]) / Number(numbers[1]),
+};
+
+const handleOperator = (type) => {
+  const operator = {
+    '+': () => calculator.plus(...state.numbers),
+    '-': () => calculator.minus(...state.numbers),
+    '*': () => calculator.multiply(...state.numbers),
+    '/': () => calculator.divide(...state.numbers),
+    '=': () => state.result,
+  };
+
+  return operator[type]();
+};
+
+function isNumber(value) {
+  return value.match(/[0-9]/g);
+}
+
+function setState(func, value) {
+  if (func(value)) {
+    state.result += value;
+  } else {
+    state.numbers.push(state.result);
+
+    if (state.operator) {
+      state.result = String(handleOperator(state.operator));
+      state.numbers.length = 0;
+      state.numbers.push(state.result);
+    }
+
+    state.operator = value;
+    state.result = '';
+  }
+}
+
+function getResult() {
+  if (state.result) {
+    return state.result;
+  }
+
+  return state.numbers[0] ? state.numbers[0] : 0;
```

Comment:
```suggestion
  return state.numbers[0] || 0;
```
short-circuit 을 이용하시면 됩니다.